packages/epics/src/treasury/components/requests/request-card.tsx

```diff
@@ -26,19 +26,13 @@ export const RequestCard: React.FC<RequestCardProps> = ({
 }) => {
   return (
     <Card className="w-full h-full p-5 mb-2 flex">
-      <Skeleton
-        loading={isLoading}
-        width="64px"
-        height="64px"
-        className="rounded-lg mr-3"
-      >
-        <PersonAvatar
-          className="min-w-[64px] min-h-[64px] mr-3"
-          avatarSrc={avatar}
-          userName={`${name} ${surname}`}
-        />
-      </Skeleton>
-      <div className="flex justify-between items-center w-full">
+      <PersonAvatar
+        size="lg"
+        isLoading={isLoading}
+        avatarSrc={avatar}
+        userName={`${name} ${surname}`}
+      />
+      <div className="flex justify-between items-center w-full ml-3">
```

Comment:
Now it's the same but different. [Check this commit](https://github.com/hypha-dao/hypha-web/pull/588/commits)